gulpfile.js

```diff
@@ -85,8 +86,19 @@ gulp.task('test', shell.task(['make test']));
 
 // Initiate a watch
 gulp.task('watch', function() {
-  gulp.watch(path.scripts, ['browserify']);
+  gulp.watch("js/**/*", ['build']);
 });
 
 // The default task (called when you run `gulp` from cli)
-gulp.task('default', ['build']);
+gulp.task('default', ['build', 'webserver', 'watch']);
```

Comment:
Was it intentional for these gulpfile changes to make it in this PR?

Reply:
Nope. Sorry. Fixed